src/UserWarning.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export const UserWarning: React.FC = () => (
```

Comment:
Consider using a more descriptive name for the component, such as 'TodoUserWarning' or 'UserIdWarning', to better convey its purpose.